portfolio.html

```diff
@@ -0,0 +1,603 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="portfolio.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <nav>
+        <div class="lower-bar">
+          <div class="haftamu">Haftamu</div>
```

Comment:
### Your design
![design (copy)](https://user-images.githubusercontent.com/93039359/179871006-7aa8302c-0577-4127-9fba-6f94c4ba0488.png)
### Figma Template
![figma](https://user-images.githubusercontent.com/93039359/179871021-af7fade8-da98-420b-8637-4448bb73cbbf.png)

- [ ] Please make your design match the Figma template. Kindly see images above and suggestions inside them :+1: